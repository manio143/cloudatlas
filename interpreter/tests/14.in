SELECT (SELECT avg(cpu_usage) WHERE false) AS smth
