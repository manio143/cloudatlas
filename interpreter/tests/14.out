/uw: smth: NULL
/pjwstk: smth: NULL
